### PR TITLE
[TECHNICAL SUPPORT]LPS-96390 JAAS login with redirect not working

### PR DESCRIPTION
**Pull request by NorbertKocsis**
https://issues.liferay.com/browse/LPS-96390

### CODE REVIEW

**Comment by NorbertKocsis** [comment]
ci:test:relevant

**Comment by topolik** [comment]
Hi @NorbertKocsis, is this LPS different from the customer reported https://issues.liferay.com/browse/LPS-95927? Thanks.

**Comment by NorbertKocsis** [comment]
Hi,

No, I just didn't know about that one as it hasn't been linked to the LPP.

Br,
Norbert

**Comment by lipusz** [comment]
ci:test:sf

**Comment by lipusz** [comment]
ci:test:relevant

File: modules/apps/login/login-web/src/main/java/com/liferay/login/web/internal/portlet/action/LoginMVCActionCommand.java [hunk 1]
```diff
@@ -272,7 +272,10 @@ protected void login(
 					return;
 				}
 
-				actionResponse.sendRedirect(mainPath);
+				HttpServletResponse httpServletResponse =
+					_portal.getHttpServletResponse(actionResponse);
+
+				httpServletResponse.sendRedirect(redirect);
```
Comment (topolik) [inline modules/apps/login/login-web/src/main/java/com/liferay/login/web/internal/portlet/action/LoginMVCActionCommand.java]: Hi @NorbertKocsis,

when is this code executed? Isn't it a dead code? Or does it redirect to "empty" redirect? This shows when the redirect is null: https://github.com/topolik/liferay-portal/blob/eed42f4b12d4b98c4d43cd738fbfab04e90d7a7b/modules/apps/login/login-web/src/main/java/com/liferay/login/web/internal/portlet/action/LoginMVCActionCommand.java#L264-L279

Thanks.

**Comment by topolik** [comment]
Pull request submitted at: https://github.com/NorbertKocsis/liferay-portal/pull/61

View just my changes: https://github.com/topolik/liferay-portal/compare/NorbertKocsis:eed42f4b12...52201f59a4
View total diff: https://github.com/topolik/liferay-portal/compare/d7a79d7aa4...52201f59a4

**Pull request closed without merging**